streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorStateManagerTest.java

```diff
@@ -520,4 +532,217 @@ public void shouldNotWriteCheckpointsIfAckeOffsetsIsNull() throws Exception {
         assertFalse(checkpointFile.exists());
     }
 
+    @Test
```

Comment:
I prefer not to use that annotation in tests with multiple lines that aren't just doing something very simple, i.e., without side-effects. I want to make sure that the exception is thrown from the path i'm testing and not some other path that happens to be in the test. In this case i also want to release the lock to clean up.